wgs.yaml

```
@@ -0,0 +1,205 @@
+sigs: []
+workinggroups:
+- dir: wg-automl
+  name: AutoML
+  mission_statement: >
+    WG AutoML is responsible for all aspects of Automated Machine Learning technologies
+    on Kubeflow.
+
+  charter_link: charter.md
+  label: area/wg-automl
+  leadership:
+    chairs:
+    - github: andreyvelich
+      name: Andrey Velichkevich
+      company: Cisco
+    - github: gaocegege
+      name: Ce Gao
+      company: Caicloud
+    - github: johnugeorge
+      name: Johnu George
+      company: Cisco
+    tech_leads:
+    - github: andreyvelich
+      name: Andrey Velichkevich
+      company: Cisco
+    - github: gaocegege
+      name: Ce Gao
+      company: Caicloud
+    - github: johnugeorge
+      name: Johnu George
+      company: Cisco
+  meetings:
+  - description: Regular Katib Meeting (Asia & Europe friendly)
+    day: Wednesday
+    time: 10:00am
+    tz: UTC (Coordinated Universal Time)
+    frequency: monthly - second Wednesday every month
+    archive_url: https://docs.google.com/document/d/1MChKfzrKAeFRtYqypFbMXL6ZIc_OgijjkvbqmwRV-64
+  - description: Regular Katib Meeting (US friendly)
+    day: Wednesday
+    time: 4:00 pm
```

I think you should also remove space from here.
```suggestion
    time: 4:00pm
```